main/templates/registration/login.html

```diff
@@ -1,11 +1,70 @@
 {% extends "base/base.html" %}
 
 {% load i18n %}
+{% load static %}
+
+{% block html_head %}
+    <script>
+        // Simple function to show emails with Javascript to keep out the bots
+        // Bots are stupid ;)
+        $(function () {
+            $('.email').each((i, e) => {
+                let element = $(e);
+                let email = element.data('email') + '@uu.nl';
+                let a = $('<a>', {
+                    href: 'mailto:' + email,
+                })
+                a.html(email)
+                // Empty the element to remove the fallback message
+                element.html('')
+                element.append(a)
+            })
+        });
+    </script>
+{% endblock %}
 
 {% block content %}
-    <div class="uu-inner-container">
-        <div class="col-12">
-            <h2>{% trans "Inloggen" %}</h2>
+    {# TODO: when migration to SAML, this template needs to become a TemplateView #}
+    <div class="coverimage">
+        <img src="{% static 'main/images/coverimage-winter.jpg' %}" />
+    </div>
+    {# Very ugly imitation of uu-hero #}
+    <div style="background: #FFCD00; margin: -10px -15px; padding:1rem;">
+        <div class="uu-inner-container" style="padding-left: 15px">
+            <h2 style="font-weight: 500">
+                {% trans "Startpagina" %}
+            </h2>
+        </div>
+    </div>
+    <div class="uu-inner-container mt-4">
+        <div class="col-12 col-md-8">
+            <p>
+                {% blocktrans trimmed %}
+                    Welkom bij het portal van de Facultaire Ethische Toetsingscommissie Geesteswetenschappen
+                    (FETC-GW).
+                {% endblocktrans %}
+            </p>
+            <p>
+                {% blocktrans trimmed %}
+                    Klik <a href="http://fetc-gw.wp.hum.uu.nl">hier</a> om terug te keren naar de FETC-GW-website.
+                {% endblocktrans %}
+            </p>
+            <h4>{% trans 'Contact' %}</h4>
+            <p>
+                <strong>{% trans 'Secretaris FETC-GW' %}</strong><br/>
+                <span class="email" data-email="fetc-gw">
+                    <i>{% trans "Om hier een e-mail adres te zien moet Javascript aan staan" %}</i>
+                </span>
+            </p>
+            <p>
+                <strong>{% trans 'Technische ondersteuning' %}</strong><br/>
+                <span class="email" data-email="portalsupport.gw">
+                    <i>{% trans "Om hier een e-mail adres te zien moet Javascript aan staan" %}</i>

```

Comment:
So thoughtful

Reply:
I wish I could take the credit; the idea is from USCKI's website lol